core/src/main/java/com/scalar/db/storage/cosmos/CosmosAdmin.java

```diff
@@ -53,7 +53,8 @@ public class CosmosAdmin implements DistributedStorageAdmin {
   public static final String DEFAULT_NO_SCALING = "false";
```

Comment:
It looks like `CosmosAdmin.databaseExists()` is no longer in use. Please remove it.